packages/client/modules/userDashboard/components/OrgMembers/OrgMembers.tsx

```diff
@@ -1,42 +1,52 @@
 import graphql from 'babel-plugin-relay/macro'
+import type {Parser as JSON2CSVParser} from 'json2csv'
+import Parser from 'json2csv/lib/JSON2CSVParser' // only grab the sync parser
 import React from 'react'
 import {PreloadedQuery, usePaginationFragment, usePreloadedQuery} from 'react-relay'
+import {OrgMembersPaginationQuery} from '~/__generated__/OrgMembersPaginationQuery.graphql'
+import {OrgMembersQuery} from '~/__generated__/OrgMembersQuery.graphql'
+import {OrgMembers_viewer$key} from '~/__generated__/OrgMembers_viewer.graphql'
+import ExportToCSVButton from '../../../../components/ExportToCSVButton'
 import Panel from '../../../../components/Panel/Panel'
-import {OrgMembersPaginationQuery} from '../../../../__generated__/OrgMembersPaginationQuery.graphql'
-import {OrgMembersQuery} from '../../../../__generated__/OrgMembersQuery.graphql'
-import {OrgMembers_query$key} from '../../../../__generated__/OrgMembers_query.graphql'
 import OrgMemberRow from '../OrgUserRow/OrgMemberRow'
 
 interface Props {
   queryRef: PreloadedQuery<OrgMembersQuery>
 }
 
-const OrgMembers = ({queryRef}: Props) => {
+const OrgMembers = (props: Props) => {
+  const {queryRef} = props
   const query = usePreloadedQuery<OrgMembersQuery>(
     graphql`
       query OrgMembersQuery($orgId: ID!, $first: Int!, $after: String) {
-        ...OrgMembers_query
+        ...OrgMembers_viewer
```

Comment:
I had refactored `OrgMembers` and `OrgMembersRoot` to use the pagination hook, but after creating this PR, a different PR was merged, which did the same refactor. The small changes here are the result of that rather than me disagreeing with the naming.